packages/help-center/src/components/help-center-contact-form.tsx

```diff
@@ -393,6 +399,29 @@ export const HelpCenterContactForm = () => {
 		}
 	};
 
+	const getSupportedLanguages = ( supportType: string, locale: string ) => {
+		switch ( supportType ) {
+			case SUPPORT_HAPPYCHAT:
+				return ! window.configData.livechat_support_locales.includes( locale );
+
+			case SUPPORT_TICKET:
```

Comment:
I am not sure if we should check all these conditions, please let me know @alshakero.

Reply:
I think the only options we need to account for are chat, email/ticket, and forum.

You are taking care of email/ticket and chat currently and you could potentially add a check against the forum locales as well if you would like? `forum_locales`?